test/integration/fixtures/blocks/core__media-text__deprecated-v5.serialized.html

```diff
@@ -1,4 +1,4 @@
-<!-- wp:media-text {"mediaType":"image","imageFill":true,"focalPoint":{"x":"0.56","y":"0.57"}} -->
+<!-- wp:media-text {"align":"wide","mediaType":"image","imageFill":true,"focalPoint":{"x":"0.56","y":"0.57"}} -->
```

Comment:
This is the result of the latest deprecation. The existing deprecation fixtures relied on the default `wide` alignment. Now it is not the default, `align` is set explicitly to `wide` and therefore stored in the block comment delimiter.